### PR TITLE
Unit test models

### PR DESCRIPTION
**Pull request by phuongha47**
## Related Tickets:
- [#45979](https://edu-redmine.sun-asterisk.vn/issues/45979)

## WHAT this PR does?
- Unit test models

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by phuongha47** [comment]
ready

**Comment by phuongha47** [comment]
ready

File: app/Models/Category.php [hunk 1]
```diff
@@ -10,9 +10,42 @@ class Category extends Model
...
+            return $this->parent->getParentsNames() . " > " . $this->name;
+        } else {
+            return $this->name;
+        }
+    }
+
+    public function getParentsAttribute()
+    {
+        $parents = collect([]);
+
```
Comment (Khoacannotcode) [inline app/Models/Category.php]: Dư xuống dòng ne

**Comment by phuongha47** [comment]
ready

**Comment by phuongha47** [comment]
ready

**Pull request merged**